src/core/User.hpp

```diff
@@ -5,8 +5,10 @@
 
 #include <QVector>
 #include <QtDebug>
+#include <optional>
```

Comment:
Not used

Reply:
I disagree. Given the [numerous fields[(https://discord.com/developers/docs/resources/user#user-object-user-flags) that could not exist when a user object is returned, a using statement is necessary.